### DISCUSSION TITLE
Unable to build nuxt 3 with graphql

### DISCUSSION THREAD

**Discussion by alnow0021** [nuxt/nuxt#19050] [Questions] [answered]
It's unable to build a nuxt 3 application with library `graphql`. During development everything is ok, there is no error, but if I run `yarn build` and later `node .output/server/index.mjs`. An error occures

```
Error 500:
Cannot find package 'D:\git\nuxt-3-graphql-repro\.output\server\node_modules\graphql\' imported from D:\git\nuxt-3-graphql-repro\.output\server\chunks\_...slugs_-c7a723bb.mjs
```
The graphql package is installed and even it's available under the path provided in error. What can be a cause of the situation?

Link to the simpliest example:
https://github.com/alnow0021/nuxt-3-graphql-repro

**Comment by TheAlexLichter** [comment, marked as answer]
Most likely you need to `transpile` the `graphql` package.

**Reply by alnow0021** [reply]
Yep, you're right. Thank you
